content/applications/inventory_and_mrp/manufacturing/management.rst

```diff
@@ -8,10 +8,10 @@ Manufacturing Management
    :titlesonly:
 
    management/bill_configuration
-   management/kit_shipping
    management/product_variants
+   management/kit_shipping
    management/sub_assemblies
+   management/quality_control
    management/alternative_wc
    management/subcontracting
-   management/routing_kit_bom
```

Comment:
Is the content of this deleted page moved elsewhere? If so, you need to create a redirection rule. See https://github.com/odoo/documentation/blob/15.0/redirects.txt